src/Native/System.Native/pal_io.cpp

```diff
@@ -443,3 +464,93 @@ int32_t MksTemps(char* pathTemplate, int32_t suffixLength)
 {
     return mkstemps(pathTemplate, suffixLength);
 }
+
+static
+void ConvertPalPollFDToNative(const PollFD& pal, pollfd& native)
+{
+    native.fd = pal.FileDescriptor;
```

Comment:
> I think we should keep doing that

Yeah, I agree.  I see the value in the more descriptive names, but that's a slippery slope, and there's definitely value in having them match as closely as possible to what's in the native functions.